File: ejercicios parte 1/1)programas simples/9que nota necesito/que_nota_necesito.cpp
#include <iostream>
using namespace std;
int main(){
    int n1, n2, lab, faltante;
    cout<<"Ingrese la nota número 1: ";
    cin>>n1;
    cout<<"Ingrese la nota número 2: ";
    cin>>n2;
    cout<<"Ingrese la nota del laboratorio: ";
    cin>>lab;

    faltante = 3*(60-lab*0.3)/0.7 - n1 - n2;
    cout<<"La nota que necesitas es: "<<faltante<<endl;

    return 0;
}

File: ejercicios parte 1/1)programas simples/9que nota necesito/que_nota_necesito.java
import java.util.Scanner;

public class que_nota_necesito {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner((System.in))) {
            int c1, c2, lab;
            double faltante;
            
            System.out.print("Ingrese la nota del primer corte: ");
            c1 = scan.nextInt();
            System.out.print("Ingrese la nota del segundo corte: ");
            c2 = scan.nextInt();
            System.out.print("Ingrese la nota del laboratorio: ");
            lab = scan.nextInt();

            faltante = 3*(60-lab*0.3)/0.7 - c1 - c2;
            System.out.println("La nota que debe sacar en el último certamen es: " + faltante);
        }
    }    
}
